docs/src/admin-manual/dashboard-config.md

```diff
@@ -56,7 +47,27 @@ institutional logo will open a new browser tab and load the given URL.
 If no `VITE_INSTITUTION_URL` is set, then the institutional logo will not be
 clickable.
 
+## OIDC settings
+
+The following environment variables can be used to configure an OpenID Connect
+(OIDC) provider for authentication and access control.
+
+```txt
+VITE_OIDC_ENABLED
+VITE_OIDC_BASE_URL
+VITE_OIDC_AUTHORITY
+VITE_OIDC_CLIENT_ID
+VITE_OIDC_SCOPES
+VITE_OIDC_ABAC_ENABLED
+VITE_OIDC_ABAC_CLAIM_PATH
+VITE_OIDC_ABAC_CLAIM_PATH_SEPARATOR
+VITE_OIDC_ABAC_CLAIM_VALUE_PREFIX
+VITE_OIDC_ABAC_USE_ROLES
+VITE_OIDC_ABAC_ROLES_MAPPING
+```
+
+Check the [Identity and access control](iac.md) page for more information.
+
```

Comment:
Looks like this page was talking about all dashboard config without mentioning these ones. I was trying to organize things a bit better to avoid duplication, but couldn't find a better compromise.

Reply:
It might be nice if you linked directly to the correct section here? i.e. `iac.md#dashboard-configuration`